om-2.0.rdf

```diff
@@ -19463,6 +19474,42 @@
     </rdfs:subClassOf>     
   </owl:Class>
 
+  <!-- Decibels Relative to Full Scale Ontology -->
+  <om:SingularUnit rdf:about="&om;decibelsRelativeToFullScale">
+    <rdfs:label xml:lang="en">dBFS</rdfs:label>
+    <rdfs:label xml:lang="nl">decibels relative to full scale</rdfs:label>

```

Comment:
Calling "decibels relative to full scale" a Dutch-language label is a bit of a stretch... I would recommend changing this to an `rdfs:comment` with `en` as language.